tests/inference/test_modeling_decoder.py

```diff
@@ -85,6 +84,8 @@ def _check_neuron_model(neuron_model, batch_size=None, num_cores=None, auto_cast
         [2, 2, "bf16"],
     ],
 )
+@is_inferentia_test
+@requires_neuronx
```

Comment:
I had some issues that the CI collects wrong tests with `parameterized` when not putting in the order. Not sure if `pytest.mark.parametrize` does better, but as the order of execution is from the bottom to the top, it will be nice if we always check if the test should be skipped or not first.